frontera/core/codec.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
 from abc import ABCMeta, abstractmethod
+import six
 
 
-class BaseDecoder(object):
-    __metaclass__ = ABCMeta
-
+class BaseDecoder(six.with_metaclass(ABCMeta, object)):
```

Comment:
I also like the [`add_metaclass` decorator way](https://pythonhosted.org/six/#six.add_metaclass) but it's nitpicking :)